pennylane/operation.py

```diff
@@ -589,21 +589,6 @@ def inverse(self):
         """Boolean determining if the inverse of the operation was requested."""
         return self._inverse
 
-    def adjoint(self, do_queue=False):
-        """Create an operation that is the adjoint of this one.
-
-        Adjointed operations are the conjugated and transposed version of the
-        original operation. Adjointed ops are equivalent to the inverted operation for unitary
-        gates.
-
-        Args:
-            do_queue: Whether to add the adjointed gate to the context queue.
-
-        Returns:
-            The adjointed operation.
-        """
-        raise NotImplementedError
```

Comment:
The reason I removed it is because it would fail my `hasattr` check. 

Although, now that I'm thinking of it, it might be better to keep this in and do a try/except instead. That way, we can either default to expand if we hit a `NotImplemented`, or raise the error if we hit anything else. I'll change it to that.

Reply:
Yeah that's nicer! I like it.